DeveloperSetupGuide.md

```diff
@@ -300,34 +300,45 @@ After rebuilding images and containers, start docker-sync normally
 ### Useful docker-sync commands
 Reference: https://docker-sync.readthedocs.io/en/latest/getting-started/commands.html
 
-## Verify DRLS is working
+## Verify the REMS prototype is working
 
 ### The fun part: Generate a test request
 
-1. Go to http://localhost:3000 and play the role of a prescriber.
+1. Go to the EHR UI at http://localhost:3000 and play the role of a prescriber.
 2. Click **Patient Select** button in upper left.
-3. Find **Jon Snow** in the list of patients and click the dropdown menu next to his name.
-4. Select **2183126 - Turalio 200 MG Oral Capsule** in the dropdown menu.
+3. Find **Jon Snow** in the list of patients and click the first dropdown menu next to his name.
+4. Select **2183126 (MedicationRequest) Turalio 200 MG Oral Capsule**  in the dropdown menu.
 5. Click anywhere in the row to select Jon Snow.
 6. Click **Send Rx to PIMS** at the bottom of the page to send a prescription to the Pharmacist.
-7. Click **Submit to REMS-Admin** at the bottom of the page.
+7. Click **Submit to REMS-Admin** at the bottom of the page, which demonstrates the case where an EHR has CDS Hooks implemented natively.
 8. After several seconds you should receive a response in the form of two **CDS cards**:
-    - **Drug Has REMS: Documentation Required.**
 9. Select **Patient Enrollment Form** on the returned CDS card with summary **Drug Has REMS: Documentation Required**.
 10. If you are asked for login credentials, use **alice** for username and **alice** for password.
 11. A webpage should open in a new tab, and after a few seconds, a questionnaire should appear.
-12. Fill out questionnaire and hit **Submit REMS Bundle**.
+12. Fill out the questionnaire and hit **Submit REMS Bundle**.
+    12a. Alternatively fill out only some of the questionnaire for an asynchronous workflow and hit **Save to EHR**.
+    12b. Visit the Patient Portal at http://localhost:3000/patient-portal and lay the role of the patient.
+    12c. Login to the Patient Portal, use **JonSnow** for the username and **jon** for the password.
```

Comment:
In Markdown, this just lists off as one big paragraph - need to figure out a way to make each sub step show up in a list